src/client.js

```diff
@@ -1,24 +1,53 @@
 /* eslint-env browser */
 import React from 'react';
+
+// Provider makes the store available to all connect() calls in the
+// component hierarchy below. (connect() is what connects a React
+// component to a Redux store.)
 import { Provider } from 'react-redux';
+
+// ReactDOM is part of isomorphic React. It renders to HTML or the DOM.
+// ex. ReactDOM.render(<MyComponent />, node)
 import ReactDOM from 'react-dom';
+
+// fromJS will convert nested Objects and Arrays to Immutable.Map and
+// Immutable.List, respectively
 import { fromJS } from 'immutable';
+
+// Router keeps the URL in sync with what's being displayed on the page
 import { Router } from 'react-router';
+
+// history abstracts away the differences in platforms and provides a
+// minimal API to manage history. we are only including the web
```

Comment:
Rather than `provides a minimal API to manage history`, what do you think about `provides a common API to manage browser history`?